charmdir.go

```diff
@@ -433,6 +433,8 @@ func (dir *CharmDir) MaybeGenerateVersionString() (string, string, error) {
 
 	vcsType := cmdArgs[0]
 	cmd := exec.Command(cmdArgs[0], cmdArgs[1:]...)
+	// We need to make sure that the wd will be the one we call the commands from.
```

Comment:
I would expand "wd" to "working directory"

Reply:
did. Thanks for the quick look !